ontology/yaml/resources/HVAC/entity_types/AHU.yaml

```diff
@@ -3574,4 +3574,25 @@ AHU_US_SVL_MAT2_4:
   - supply_fan_run_command
   - supply_fan_speed_frequency_sensor
   - supply_fan_speed_percentage_command
-  - supply_air_flowrate_sensor
\ No newline at end of file
+  - supply_air_flowrate_sensor
+
+AHU_STDSPC_SSPC_EFFC_FDPM3X_RTM_RHM:
```

Comment:
We should attempt to model these entities as incomplete versions of a type, since a lot of the data is missing. I'd expect SFSS, SFVSC, EFSS, EFVSC, and so on. Is the heat wheel the only non-standard part of this?

Reply:
@tasodorff heat wheel and supply_air_isolation_damper_status + exhaust_air_isolation_damper_status. Should those be abstract types too?

Reply:
if heat wheel comes with run_command, run_status, and speed_percentage_command as stock, then we would create that as a type. probably with fields such as these:

- heat_wheel_run_command
- heat_wheel_run_status
- heat_wheel_speed_percentage_command

exhaust_air_isolation_damper_status/exhaust_air_isolation_damper_command should be added as their own abstract type, if they don't exist already.